added export script for tf 1.14 using TFLiteConverter and relu6 fix.

PROGrand opened the pull request.

PROGrand commented:
I signed it!

jdduke commented:
Can you provide a little more context for this? What exactly is the issue you're facing?

PROGrand commented:
I use standard tensorflow 1.14 setup, with pip install tensorflow-gpu, keras and etc.
I have trained custom audio recognition model with train.py. But i was unable to convert keras model file to tflite because of relu6 error. I fixed relu6 and switched from deprecated TocoConverter to TFLiteConverter.

jdduke commented:
Do you have a link to the model which fails to convert? It's not clear why nn.relu6 isn't used in this pipeline.

PROGrand commented:
https://drive.google.com/open?id=1-Q1Djuw3CE2sIQc0og-FEFM-uy55hNwv

jdduke commented:
Thanks @PROGrand, that's very helpful. Let me sync internally with some folks to see if we need to land a more general fix in our converter for all Keras models.

PROGrand commented:
relu6 problem for reference: https://github.com/tensorflow/tensorflow/issues/17191#issuecomment-442532008

Qiuzhuang commented:
I get this error and validate this patch works. Thanks.

MarkDaoust commented:
Hi,

I think this is fixed now. The problem was in how that model was being constructed.

Old versions of `keras.applications` has a custom [activation function there](https://github.com/keras-team/keras-applications/blob/1.0.1/keras_applications/mobilenet.py#L77).

But now [the code](https://github.com/keras-team/keras-applications/blob/master/keras_applications/mobilenet.py#L361) uses [layers.ReLU](https://www.tensorflow.org/api_docs/python/tf/keras/layers/ReLU) with a max-value argument.
